src/static/js/app/components/Survey.js

```diff
@@ -11,6 +11,9 @@ export default class Survey extends React.Component {
     questions: [],
     nextPage: 2
   };
+  static defaultProps = {
```

Comment:
Expected '(' and instead saw 'defaultProps'.<br>Expected an identifier and instead saw '='.<br>Expected ')' to match '{b}' from line 14 and instead saw '{'.<br>'function closure expressions' is only available in Mozilla JavaScript extensions (use moz option).